File: year2020/month10/day1013/ms_1013.cs
using System;

namespace algorithm_77
{
    class ms_1013
    {
        static int Recursion(int n) 
        {
            if(n == 1)
                return 1;
            else if(n == 2)
                return 2;
            else
                return (Recursion(n-2) * Recursion(n-1)) % 100;
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(0 < n && n <= 100)
                Console.WriteLine(Recursion(n));
        }
    }
}

File: year2020/month10/day1013/ms_1013.cpp
#include <iostream>

using namespace std;

int recursion(int n);

int main() {
	int n;

	cin >> n;

	if (0 < n && n <= 100)
		cout << recursion(n) << endl;

	return 0;
}

int recursion(int n) {
	if (n == 1)
		return 1;
	else if (n == 2)
		return 2;
	else
		return (recursion(n - 2) * recursion(n - 1)) % 100;
}